bin/mmdblookup.c

```diff
@@ -396,3 +431,105 @@ LOCAL void random_ipv4(char *ip)
     snprintf(ip, 16, "%u.%u.%u.%u",
              *bytes, *(bytes + 1), *(bytes + 2), *(bytes + 3));
 }
+
+#ifndef _WIN32
+struct thread_info {
+    pthread_t id;
+    int num;
+    MMDB_s *mmdb;
+    int iterations;
+};
+#endif
+
+LOCAL bool start_threaded_benchmark(
+    MMDB_s *const mmdb,
+    int const thread_count,
+    int const iterations)
+{
+#ifdef _WIN32
+    return false;
```

Comment:
Why not just remove the function definitions for these on Windows altogether and remove the following?
 https://github.com/maxmind/libmaxminddb/blob/ec604d71879583604d8efeae2d288474d710bf01/bin/mmdblookup.c#L82-L89

Reply:
Yeah, that's nicer. Mainly I was trying to work around `regen-prototypes.pl` not being happy with macros (it deletes them from its section). I found a different workaround that seems better, although is still a bit ugly what with the extra `NO_PROTOTYPE` macro. I wonder if we need `regen-prototypes.pl`. It's not a huge pain to keep them up to date manually and would avoid this problem.

Reply:
Yeah, I would be fine getting rid of that script. I don't find it particularly useful.